build.gradle.kts

```diff
@@ -127,6 +158,8 @@ tasks {
             // Eliminate dependencies' pom files
             exclude("**/pom.*")
         }
+        // We can't copy the manifest for some reason, but we can copy all the attributes of the manifest.
```

Comment:
I'd like to know more about this, but okay.

Reply:
Oh, so I figured it out. You can't call `manifest.inheritFrom(someOtherTask)`, but you can call `manifest.inheritFrom(someOtherManifest)`. I missed that because the `inheritFrom` method accepts var args `Object` so there's no help from the method signature. After trying a few incorrect things, I found that copying the manifest attributes would work.

I'll update this.